packages/ui/primitives/combobox.tsx

```diff
@@ -16,66 +15,61 @@ import { Popover, PopoverContent, PopoverTrigger } from '@documenso/ui/primitive
 
 type ComboboxProps = {
   listValues: string[];
-  onChange: (_values: string[]) => void;
+  onChange: (_value: string) => void;
+  selectedValue: string | null;
+  disabled?: boolean;
 };
 
-const Combobox = ({ listValues, onChange }: ComboboxProps) => {
+const Combobox = ({ listValues, onChange, selectedValue, disabled = false }: ComboboxProps) => {
   const [open, setOpen] = React.useState(false);
-  const [selectedValues, setSelectedValues] = React.useState<string[]>([]);
-  const dbRoles = Object.values(Role);
+  const [selectedValueLocal, setSelectedValueLocal] = React.useState<string | null>(selectedValue);
 
   React.useEffect(() => {
-    setSelectedValues(listValues);
-  }, [listValues]);
-
-  const allRoles = [...new Set([...dbRoles, ...selectedValues])];
+    setSelectedValueLocal(selectedValue);
+  }, [selectedValue]);
 
   const handleSelect = (currentValue: string) => {
-    let newSelectedValues;
-    if (selectedValues.includes(currentValue)) {
-      newSelectedValues = selectedValues.filter((value) => value !== currentValue);
-    } else {
-      newSelectedValues = [...selectedValues, currentValue];
-    }
-
-    setSelectedValues(newSelectedValues);
-    onChange(newSelectedValues);
+    setSelectedValueLocal(currentValue === selectedValueLocal ? null : currentValue);
+    onChange(currentValue === selectedValueLocal ? '' : currentValue);
     setOpen(false);
   };
 
   return (
-    <Popover open={open} onOpenChange={setOpen}>
-      <PopoverTrigger asChild>
-        <Button
-          variant="outline"
-          role="combobox"
-          aria-expanded={open}
-          className="w-[200px] justify-between"
-        >
-          {selectedValues.length > 0 ? selectedValues.join(', ') : 'Select values...'}
-          <ChevronsUpDown className="ml-2 h-4 w-4 shrink-0 opacity-50" />
-        </Button>
-      </PopoverTrigger>
-      <PopoverContent className="w-[200px] p-0">
-        <Command>
-          <CommandInput placeholder={selectedValues.join(', ')} />
-          <CommandEmpty>No value found.</CommandEmpty>
-          <CommandGroup>
-            {allRoles.map((value: string, i: number) => (
-              <CommandItem key={i} onSelect={() => handleSelect(value)}>
-                <Check
-                  className={cn(
-                    'mr-2 h-4 w-4',
-                    selectedValues.includes(value) ? 'opacity-100' : 'opacity-0',
-                  )}
-                />
-                {value}
-              </CommandItem>
-            ))}
-          </CommandGroup>
-        </Command>
-      </PopoverContent>
-    </Popover>
+    <div className="flex max-h-[100px] items-center space-x-4">
```

Comment:
What is this additional div for?

Reply:
Had added it while I was playing around with the styles, have remvoed it :)